packages/browser/services/mongodb-remote/src/RemoteMongoCollection.ts

```diff
@@ -98,6 +98,19 @@ export default interface RemoteMongoCollection<DocumentT> {
     options?: RemoteFindOptions
   ): RemoteMongoReadOperation<DocumentT>;
 
+  /**
+   * Finds one document in the collection that matches the given query.
+   * 
+   * An empty query (`{}`) will match all documents.
+   *
+   * @param query the query filter
+   * @return the resulting document or undefined if the query resulted in zero matches
+   */
+  findOne(
+    query?: object,
+    options?: RemoteFindOptions
+  ): Promise<DocumentT | undefined>;
```

Comment:
[minor] if `findOne` is returning null, the type here should be `Promise<DocumentT | null>`, but we should have a discussion in Slack about that